### PR TITLE
💄#357: update tooltips

### PR DESCRIPTION
**Pull request by xusandrew**
## Description of Changes

gets rid of the current tooltips we have, and replaced them with small text that is always visible. 

## PR Readiness Checklist

- [x] "Expected Outcome(s)" in ticket have been met
- [x] Ticket number included in PR title
- [x] Connected ticket to PR
- [x] Labels added to PR for service name (`consent-api`, `data-mapper`, etc...), type (`chore`, `documentation`, etc...), status (`draft`, `on-hold`, etc...) **if applicable**
- [x] Manual testing completed
- [x] Builds locally without errors or warnings
- [x] Tests are updated (if required) and passing
- [ ] PR feedback has been addressed **if applicable**
- [x] New environment variables added to `.env.schema` files, `README.md`
- [x] Added copyrights to new files

### CODE REVIEW

**Review by mistryrn — COMMENTED** [review]
Nice and clean! 🙂 Looks good to me, but I'll defer to Sam for final approval since she build the original Tooltip implementation

File: apps/consent-ui/src/components/common/Form/fieldsets/CalendarFieldSet/index.tsx [hunk 1]
```diff
@@ -48,8 +47,6 @@ const CalendarFieldSet = <T extends FieldValues>({
 	withNarrowDesktopLayout,
 	currentLang,
 }: CalendarFieldSetProps<T>) => {
-	const { tooltipVisible, hideTooltip, showTooltip } = useTooltip();
-
 	const idPrefix = useId();
 	const fieldId = `${idPrefix}-${name}`;
 	const tooltipId = `${idPrefix}-tooltip`;
```
Comment (samrichca) [inline apps/consent-ui/src/components/common/Form/fieldsets/CalendarFieldSet/index.tsx:52]: looks great so far but I'd rename tooltip to description - it's not a tooltip anymore

File: apps/consent-ui/src/components/common/Form/fieldsets/CalendarFieldSet/index.tsx [hunk 2]
```diff
@@ -48,8 +47,6 @@ const CalendarFieldSet = <T extends FieldValues>({
 	withNarrowDesktopLayout,
```
Comment (samrichca) [inline apps/consent-ui/src/components/common/Form/fieldsets/CalendarFieldSet/index.tsx:47]: withNarrowDesktopLayout was added to adjust the tooltip placement on pages with a sidebar (invite & register), please remove it 👍

**Review by samrichca — APPROVED** [review]
perfect, thanks for making those other changes

**Pull request merged**